books/RayTracingInOneWeekend.html

```diff
@@ -4378,6 +4377,7 @@
 [gfx-codex]:          https://graphicscodex.com/
 [releases]:           https://github.com/RayTracing/raytracing.github.io/releases/
 [repo]:               https://github.com/RayTracing/raytracing.github.io/
+[compiling]:          https://github.com/RayTracing/raytracing.github.io?tab=readme-ov-file#building-and-running
```

Comment:
When referring to the project readme, prefer the local copy, as people can download and use the material offline, and the current version may lack information or have different information that once applied to an older version.

```
[readme]: ./README.md
```